src/routes/Trader/Trader.tsx

```diff
@@ -43,10 +43,13 @@ const Trader: React.FunctionComponent = () => {
 
   const referrerKey = 'invitedBy';
   const [searchParams] = useSearchParams();
-  if (!!searchParams.get(referrerKey) && !localStorage.getItem(referrerKey)) {
+  if (
+    (!!searchParams.get(referrerKey) && !localStorage.getItem(referrerKey)) ||
+    localStorage.getItem(referrerKey)?.length !== 8 // Earlier, pre-release keys may have used more than 8 characters but we overwrite these
+  ) {
     // Referrer is set in URL and has not already been saved in storage; save to storage now
     // We do deliberately do not overwrite any existing value because we want to credit the first referral link that was followed
-    localStorage.setItem('invitedBy', searchParams.get('invitedBy') || '');
+    localStorage.setItem(referrerKey, searchParams.get(referrerKey) || '');
```

Comment:
should we have a function to construct the referrerKey? I think this key should depend on "env" where testing happens. What if someone is testing on test.voltz.xyz and we set stuff to localStorage. Then they go on real app.voltz.xyz and they get invite.... Because they are part of our test group the app wont for them on production.

Reply:
I think localStorage is per domain and port, so values set on testnet won't be read on mainnet and vice versa. Please shout if I've misunderstood.

Reply:
thakns for sharing this!